test/common/stats/lazyinit_stats_speed_test.cc

```diff
@@ -0,0 +1,219 @@
+#include "source/common/common/random_generator.h"
+#include "source/common/stats/isolated_store_impl.h"
+#include "source/common/stats/lazy_init.h"
+#include "source/common/stats/symbol_table.h"
+#include "source/common/stats/thread_local_store.h"
+#include "source/common/thread_local/thread_local_impl.h"
+#include "source/exe/process_wide.h"
+
+#include "test/benchmark/main.h"
+#include "test/common/stats/real_thread_test_base.h"
+#include "test/test_common/real_threads_test_helper.h"
+
+#include "benchmark/benchmark.h"
+
+namespace Envoy {
+namespace Stats {
+
+// Creates a copy of Upstream::ALL_CLUSTER_TRAFFIC_STATS, such that per Joshua, we have a "a stable
```

Comment:
Can we de-personalize this?

It's fine to note that the source for these particular names (though really the names don't matter that much you could've called them counter1 counter2 etc.

And I think there's no need to keep them up to date with the original one.